index.html

```diff
@@ -38,6 +39,131 @@ <h1 class="primary-text">I’m Richmond Glad to see you!</h1>
             <li class="angelistic"><a href="#"><img src="./logos/angelist.png" alt="github-logo" /></a>
             </li>
         </ul>
+      </section>
+      <section class="works-container">
+        <div class="projects-cards">
+          <div class="card">
+          <div class="img"><img src="./logos/t2.png" alt="work example" /></div>
+            <p class="card-title">Tonic</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+              </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>Javascript</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+          <div class="card">
+            <div class="img"><img src="./logos/t5.png" alt="work example" /></div>
+            <p class="card-title">Multi-Post Stories</p>
+            <div class="tag">
+              <ul>
+                <li>Canopy</li>
+                <li>Backend-Dev</li>
+                <li> 2015</li>
+            </ul>
+             </div>
+            <p class="card-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="button1"><a href="#">See Project</a></div>
+          </div>
+          <div class="card">
+            <div class="img"><img src="./logos/t3.png" alt="work example" /></div>
+            <p class="card-title">Tonic</p>
+            <div class="tag">
```

Comment:
- [ ] The project title (e.g., "Tonic") should use a header tag `hx`(`<h2>`, `<h3>` etc) instead of `p` tag.